.travis.yml

```diff
@@ -67,8 +68,7 @@ after_success:
   # Copy compiled wheels to dist/ where Travis `dpl` tool can find them and 
   # upload to PyPI
   - if [ -n "$TRAVIS_TAG" ]; then mkdir -p dist; cp wheelhouse/*.whl dist; fi
-  # Additionally build sdist on tag and not bundling.
-  - if [ -z $"FREETYPEPY_BUNDLE_FT" ] && [ -n "$TRAVIS_TAG" ]; then python setup.py sdist; fi
+  - if [ -n "$BUILD_SDIST" ]; then python setup.py sdist; fi
```

Comment:
you also need to check that it is a tagged commit otherwise it will run for every commit, not only when `[ -n "$TRAVIS_TAG" ]`

Reply:
Dammit 😁